modules/ombi.py

```diff
@@ -0,0 +1,540 @@
+
+    @cherrypy.expose()
+    @require(member_of(htpc.role_admin))
+    def content_sync(self, source, mode, **kwargs):
+        if source == 'plex':
+            if self.get_plex_enabled() == 'True':
+                if mode == 'full':
+                    url = 'api/v1/Job/plexcontentcacher'
+                else:
+                    url = 'api/v1/Job/plexrecentlyadded'
+        if source == 'emby':
+            if self.get_emby_enabled() == 'True':
+                url = 'api/v1/Job/embycontentcacher'
+        d = self._ombi_post(url)
+        if d != 'False':
+            logger.debug('%s %s sync triggered' % (source, mode))
+            return 'True'
+        logger.debug('%s %s sync failed' % (source, mode))
+        return 'False'
+
+    @cherrypy.expose()
+    @require()
+    def get_emby_enabled(self):
+        d = self._ombi_get('api/v1/Settings/emby')
+        if d != 'False':
+            return str(d['enable'])
+        logger.debug('Couldnt get Ombi settings for Emby')
+        return 'False'
+
+    @cherrypy.expose()
+    @require()
+    def get_plex_enabled(self):
+        d = self._ombi_get('api/v1/Settings/plex')
+        if d != 'False':
+            return str(d['enable'])
+        logger.debug('Couldnt get Ombi settings for Plex')
+        return 'False'
+
+    def _ombi_get(self, url):
+        """
+        Combined function to make all swagger API GET calls.
+        Builds the url, authenticates and grabs token if there isn't one.
+        :url: api endpoint
+        :return: the json response content or 'False'
+        """
+        if self.auth() == 'False':
+            logger.error('GET request died - auth failed')
+            return 'False'
+        ssl = 's' if htpc.settings.get('ombi_ssl', False) else ''
+        ip = htpc.settings.get('ombi_host')
+        port = htpc.settings.get('ombi_port')
+        u = 'http%s://%s:%s/%s' % (ssl, ip, port, url)
+        h = dict()
+        h.update({ 'Authorization': self._token})
+        r = requests.get( u, headers=h )
+        if r.status_code == 200:
+            d = r.json()
+            return d
+        logger.error('Request failed %s %s %s' % (u, str(r.status_code), r.reason))
+        return 'False'
+
+    def _ombi_post(self, url, data=''):
+        """
+        Combined function to make all swagger API POSTs.
+        Builds the url, authenticates and grabs token if there isn't one.
+        :url: api endpoint
+        :param data: post data in json format
+        :return: the response data in json format or 'False'
+        """
+        logger.debug('Doing POST request to %s:\n%s' % (url, data) )
+        if self.auth() == 'False':
+            logger.error('POST request died - auth failed')
+            return 'False'
+        ssl = 's' if htpc.settings.get('ombi_ssl', False) else ''
+        ip = htpc.settings.get('ombi_host')
+        port = htpc.settings.get('ombi_port')
+        u = 'http%s://%s:%s/%s' % (ssl, ip, port, url)
+        h = dict()
```

Comment:
remove the extra whitespace or use the dict directly
```
h = {'Accept': 'application/json',
      .....
}
```

Reply:
I am not sure what you mean with this comment.  Are you referring to the several .update() calls in a row, rather than a single { xxx: yyy, jjj: kkk } multi-value dict update?
What whitespace are you referring to?

Reply:
Done